vendor/modules.txt

```diff
@@ -182,7 +182,9 @@ github.com/json-iterator/go
 github.com/kubevirt/monitoring/pkg/metrics/parser
 # github.com/machadovilaca/operator-observability v0.0.13
 ## explicit; go 1.21
+github.com/machadovilaca/operator-observability/pkg/docs
```

Comment:
Is it planned to release it using `vendor/github.com/machadovilaca/operator-observability` from @machadovilaca account?

Reply:
@codingben It is planned to move to kubevirt org in the near future